pylib/mozhginfo/mozhginfo/push.py

```diff
@@ -0,0 +1,62 @@
+class Push(object):
+    """This represents a Push"""
+
+    def __init__(self, date):
```

Comment:
My suggestion was to use a dict that could expand without having to worry about the `Push` object having to grow and then we get a very unwieldy `__init__` which is poor API design. Since the creation of this object should be handled internally and then passed back the consumer it shouldnt really matter (as long as it is documented)